docs/hardware.md

```diff
@@ -86,7 +86,7 @@ Displaying this 64x32 image will have roughly the same cost as displaying eight
 
 In summary:
 
-1. There's a hard limit of 80 sprites on screen and a total of 56kB active sprite data
+1. There's a hard limit of 80 sprites on screen and a total of 54kB active sprite data
```

Comment:
I ran out of RAM on the GPU so stole 2kB from the sprite data - though I suspect you can't actually fill this data buffer during the available VSYNC time anyway.